pkg/cli/plugin_cmd.go

```diff
@@ -26,6 +26,7 @@ func GetCmdForPlugin(p *PluginInfo) *cobra.Command {
 		DisableFlagParsing: true,
 		Annotations: map[string]string{
 			"group": string(p.Group),
+			"scope": p.Scope,
```

Comment:
This was added to TF after we moved it to this repo:
https://github.com/vmware-tanzu/tanzu-framework/blob/8354312f13e95a96ccee234d3423ede33eeec09c/cli/core/pkg/cli/cmd.go#L31